cmds/core/mount/mount.go

```diff
@@ -28,6 +30,11 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+var (
+	errUsage     = errors.New("usage")
+	errMountPath = errors.New("can't read mount path")
```

Comment:
I think we're supposed to avoid contractions in error messages?
can't read mount path
can not read mount path

Reply:
updated